### DISCUSSION TITLE
Add a server time feature to Discord GSA bot for 7 Days to Die

### DISCUSSION THREAD

**Discussion by ronnocnave** [gameserverapp/Platform#121] [Suggestions]
In the current GSA Discord bot command list, there is no way to get the server day / time. Nor is there a way to get this info from the Gameserverapp.com dashboard. Since 7 Days is very day / time -centric because of the nature of "hoard nights", a way to get the current server time without logging in would be MOST helpful! Something like !time would do the trick.

**Comment by mvaessen** [comment]
Hi, do you know whether this information is available via RCON or Query ports?

**Comment by ronnocnave** [comment]
Should be, via the !gettime command.

**Comment by DieselEffect** [comment]
Did this ever get any traction? Having the in-game Day # and HH:MM time would be exceptionally useful in discord. Even if it was just displayed in the Server Status card.
